src/Core/src/Platform/iOS/MauiTextField.cs

```diff
@@ -18,10 +19,18 @@ public MauiTextField()
 		{
 		}
 
+		[UnconditionalSuppressMessage(IUIViewLifeCycleEvents.UnconditionalSuppressMessage, "MA0002")]
+		EventHandler? _movedToWindow;
+		event EventHandler IUIViewLifeCycleEvents.MovedToWindow
+		{
+			add => _movedToWindow += value;
+			remove => _movedToWindow -= value;
+		}
+
 		public override void WillMoveToWindow(UIWindow? window)
 		{
 			base.WillMoveToWindow(window);
-			ResignFirstResponderTouchGestureRecognizer.Update(this, window);
+			_movedToWindow?.Invoke(this, EventArgs.Empty);
 		}
```

Comment:
Is this correct? The WillMove uses the Moved event? Maybe we should just have both on the interface?

Reply:
fixed